pkg/smartcontract/manifest/standard/check.go

```diff
@@ -12,4 +12,6 @@ type Standard struct {
 	// If contract contains method with the same name and parameter count,
 	// it must have signature declared by this contract.
 	Optional []manifest.Method
+	// Required contains standards that are required for this standard.
+	Required []string
```

Comment:
Move `Required`-related changes to a separate commit, because it's a separate logical change that in fact is not connected with royalties.